### PR TITLE
added products page

### PR DESCRIPTION
**Pull request by tusharkandpal**
- products html consists of: navbar, sidebar & products
- stylesheet for product page

### CODE REVIEW

File: components/product-page/products.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  display: inline-block;
+  margin-top: 1rem;
+}
+.item-input {
+  margin-right: 0.3rem;
+}
+
+/* ---------------------- || PRODUCTS SECTION --------------------------------*/
+.products-section {
+  background-color: #f5f5f5;
```
Comment (shaksham08) [inline components/product-page/products.css:39]: use color variable

**Review by shaksham08 — APPROVED** [review]
Looks good , just see the comments

File: components/product-page/products.css (hunk 1 above)
Comment (tusharkandpal) [reply components/product-page/products.css:39]: ya that i will do once i finish this site.

**Review by Bhakti199 — APPROVED** [review]
good to merge

**Pull request merged**